src/worker.js

```diff
@@ -331,12 +332,27 @@ self.onmessage = (e) => {
       }
       constraint.uuid = uuid
       state.world.addConstraint(constraint)
+
+      const postStepConstraint = () => {
+        if (isNaN(maxMultiplier)) {
+          return
+        }
+        // The multiplier is proportional to how much force that is added to the bodies by the constraint.
+        // If this exceeds a limit we disable the constraint.
+        const multiplier = Math.abs(constraint.equations[0].multiplier)
+        if (multiplier > maxMultiplier) {
+          constraint.disable()
+        }
+      }
+      state.constraints[uuid] = postStepConstraint
+      state.world.addEventListener('postStep', state.constraints[uuid])
```

Comment:
I don't think we should add a postStep handler for every constraint. It should only for the ones that have a maxMultiplier.

Reply:
Will change. Thanks